Parent/Parent/Conversations/ConversationCoursesActionSheet.swift

```diff
@@ -0,0 +1,106 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2020-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import Core
+import CoreData
+
+protocol ConversationCoursesActionSheetDelegate: class {
+    func courseSelected(course: Course, user: User)
+}
+
+class ConversationCoursesActionSheet: UITableViewController, ErrorViewController {
+    let env: AppEnvironment = .shared
+    weak var delegate: ConversationCoursesActionSheetDelegate?
+    let loadingIndicator = UIActivityIndicatorView(frame: CGRect(x: 0, y: 0, width: 40, height: 40))
+
+    lazy var enrollments = env.subscribe(GetConversationCourses()) { [weak self] in
+        self?.update()
+    }
+
+    static func create(delegate: ConversationCoursesActionSheetDelegate) -> ConversationCoursesActionSheet {
+        let vc = ConversationCoursesActionSheet()
+        vc.modalPresentationStyle = .custom
+        vc.transitioningDelegate = ActionSheetTransitioningDelegate.shared
+        vc.delegate = delegate
+        return vc
+    }
+
+    override func viewDidLoad() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.separatorStyle = .none
+        tableView.registerCell(SubtitleTableViewCell.self)
+
+        let header = UILabel(frame: CGRect(x: 16, y: 0, width: tableView.frame.width - 32, height: 40))
```

Comment:
This header looks a hair too small. In figma it's 40 high but has a top padding of 8. if you make the y: 8 will that work? I wonder if the y will get ignored...